samples/bidirectional-chat/csharp/Authorize/FunctionAuthorizeAttribute.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Azure.WebJobs.Extensions.SignalRService;
+using Microsoft.Azure.WebJobs.Host;
+
+namespace bidirectional_chat
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
+#pragma warning disable CS0618 // Type or member is obsolete
+    internal class FunctionAuthorizeAttribute: FunctionInvocationFilterAttribute
+    {
+        private const string AdminKey = "admin";
+
+        public override Task OnExecutingAsync(FunctionExecutingContext executingContext, CancellationToken cancellationToken)
```

Comment:
Can we extract some common logic into a base class so that developer only needs to focus on the claim validation part.

Reply:
That's good. I will use another PR to do that.